resources/docker-build_p2p/start_x11vnc.sh

```diff
@@ -49,8 +49,15 @@ do
     esac
 done
 
+if [ "$1" = '' ]
+then
+    usage
+fi
+
+stunDumpFileLoaction="$1";
```

Comment:
is this actually just the file's location or is it the path with the file name as well? I think it should be a bit clearer, in port_share it says "stun dump file" and here it says "stundumpfilelocation"